packages/stylelint-config/index.js

```diff
@@ -1,47 +1,27 @@
 module.exports = {
   customSyntax: 'postcss-scss',
+  extends: [
+    "stylelint-config-standard-scss",
+    "stylelint-config-recommended"
+  ],
   rules: {
-    'block-no-empty': true,
-    'string-no-newline': true,
-    'string-quotes': 'single',
-    'unit-no-unknown': true,
+    // Overridden rules from `stylelint-config-recommended`
+    "at-rule-no-unknown": null,
```

Comment:
It would also be great if you could add a link and some explanation of what the rule does or why they differentiate from the community standard. Please see https://github.com/lmc-eu/code-quality-tools/blob/main/packages/eslint-config-react/rules/react.js for example. But this is nice to have and I can do the issue for this.

Reply:
Not so precisely done yet but please check it out now again. I have added explanation to each rule which values are not clear at first look.

Reply:
Conversation solved by internal discussion.